pkg/logs/agent_test.go

```diff
@@ -74,19 +74,19 @@ func (suite *AgentTestSuite) TearDownTest() {
 
 func createAgent(endpoints *config.Endpoints) (*Agent, *sources.LogSources, *service.Services) {
 	// setup the sources and the services
-	sources := sources.NewLogSources()
+	logsSources := sources.NewLogSources()
```

Comment:
I'm not sure why this name was changed, but the double "s" isn't used -- it's LogSource, not LogsSource.